tests/integration/up_down/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+volumes:
+    web1_vol:
+    web2_vol:
+services:
+    web1:
+      image: podman-compose-up-down-test
+      build: .
+      hostname: web1
+      command: ["dumb-init", "/bin/busybox", "httpd", "-f", "-h", "/var/www/html", "-p", "8001"]
```

Comment:
Better launch `sleep infinity` and not add unnecessary moving parts. Stop commands will need `-t0` in such case.